pkg/connector/connector.go

```diff
@@ -156,6 +170,13 @@ func newGithubClient(ctx context.Context, instanceURL string, accessToken string
 
 // New returns the GitHub connector configured to sync against the instance URL.
 func New(ctx context.Context, githubOrgs []string, instanceURL, accessToken string) (*Github, error) {
+	// TODO: add more robust way of handling enterprise environements (cloud enterprise have same URL as normal or team plan)
+	isEnterprise := false
+	instanceURL = strings.TrimSuffix(instanceURL, "/")
+	if instanceURL != "" && instanceURL != githubDotCom {
+		isEnterprise = true
+	}
```

Comment:
We could probably normalize the URL using url.parse() (which would also let us error earlier if the url is invalid), then check that the hostname is github.com.